packages/create-app/src/prompts/init.ts

```diff
@@ -152,6 +157,9 @@ const init = async (options: InitOptions): Promise<InitOutput> => {
 
   answers.template = selectedUrl || answers.template || defaults.template
 
+  const installed = await installGlobalCLIIfNeeded(options.packageManager)
```

Comment:
This will install the global CLI right? It feels a bit weird that a prompt has such a side-effect